src/autogluon/cloud/scripts/image_serve.py

```diff
@@ -12,12 +12,15 @@
 from autogluon.core.utils import get_pred_from_proba_df
 from autogluon.vision import ImagePredictor
 
+image_dir = os.path.join("/tmp", "ag_images")
 
-def _cleanup_images():
-    files = os.listdir(".")
-    for file in files:
-        if file.endswith(".png"):
-            os.remove(file)
+
+def _save_images(im, im_name):
+    os.makedirs(image_dir, exist_ok=True)
+    im_path = os.path.join(image_dir, im_name)
+    im.save(im_path)
+
+    return im_path
```

Comment:
Can we make this as an shared util function so that `tabular_serve.py` can also use?

Reply:
No. This are python files that will be sent to the container. Nothing can be shared

Reply:
Got it. I like that you are making this as a function, maybe we can do the same in `tabular_serve.py`.